packages/engine/tests/units/topology/distance/experiments.json

```diff
@@ -0,0 +1,8 @@
+{
+  "test": {
+    "type": "values",
+    "steps": 2,
+    "field": "topology.distance_function",
+    "values": ["euclidean_squared", "manhattan", "euclidean", "conway"]
```

Comment:
For most simulations we agreed to create separate experiments for each test. I guess for these we don't think that's as much of a concern? What happens if we do break the "manhattan" distance for example, will we still get results for euclidean and conway?

Reply:
No, we won't. I can duplicate the simulations and make them separate tests.

Reply:
Yeah lets duplicate